lldb/test/API/functionalities/inline-stepping/TestInlineStepping.py

```diff
@@ -32,6 +32,12 @@ def test_step_in_template_with_python_api(self):
         self.build()
         self.step_in_template()
 
+    @add_test_categories(["pyapi"])
+    def test_virtual_inline_stepping(self):
+        """Test stepping through a virtual inlined call stack"""
+        self.build()
+        self.virtual_inline_stepping()
+
     def setUp(self):
```

Comment:
I believe this entire function is no longer needed.

Reply:
Not sure what you mean.  This is the style we often write tests in where we have the to unittest significant `test_whatever` function be just a shim on the real test function.  That makes it easy to see what all the tests are up front without having to wade through the details.  That's how all the other tests in this file work as well.

Reply:
I was referring to the setUp() function that just calls the parent setUp function. If it's still needed ignore this,.

Reply:
Ah.  The other tests still need the setUp.  I don't use need the work it does for the test I added, but it doesn't do very much, so IMO it's not worth making another test class just to avoid calling it.